src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -1341,6 +1341,65 @@
         az acr token update -n MyToken -r MyRegistry --scope-map MyNewScopeMap
 """
 
+helps['acr artifact-streaming'] = """
+type: group
+short-summary: Manage artifact streaming for any repositories or supported images in an ACR.
+"""
+
+helps['acr artifact-streaming operation'] = """
+type: group
+short-summary: Allows you to manage the streaming artifact creation for ACR.
+"""
+
+helps['acr artifact-streaming operation cancel'] = """
+type: command
+short-summary: Cancels the given streaming artifact operation for ACR.
```

Comment:
It is suggested to use first-person voice in help message.

Reply:
I'll be updating the rest as well